### PR TITLE
Fix master

### PR DESCRIPTION
**Pull request by bryan-robitaille**
**Issue:**  Session Cookie not being properly set through docker config.
**Reason:**  `os.getenv()` can only return a string.  Some Django settings require an explicit boolean.
**Solution:**  Created a function to wrap `os.getenv()` in when calling a setting that requires an explicit boolean from docker-compose.

**Issue:**  docker-compose settings not being activated in docker container
**Reason:** Dockerfile logic when copying files from project to container faulty.  If an existing  `pleio_account/pleio_account/config.py` the Dockefile logic will not overwrite it with the correct docker config.py.
**Solution:**  Inverse the copying operation from root to container.  Copy all files then perform the config.py substitution.

**Issue:**  Email Validator Migration 003 failing
**Reason:**  Migration 002 creates Null fields on existing rows and 003 modifies to explicitly be non-Null without providing a default value.
**Solution:**  Modify migration to use a temporary default value without modifying the defined model.

**Issue:**  `SEND_SUSPICIOUS_BEHAVIOR_WARNINGS` config.py setting is no longer functioning
**Reason:** Setting is now derived from "Receives Newsletter" on user model.
**Solution:**  Remove setting from config.py for now but require refactor to not cause development confusion in future coding.

**Issue:** Language files not compiling in start.sh when container is launched.
**Reason:**  Dockerfile missing GNU tools in last docker image layer
**Solution:**  Add `apk add gettext` to last image layer creation for container

### CODE REVIEW

File: docker/config.py [hunk 1]
```diff
@@ -23,8 +24,6 @@
 
 STATIC_ROOT = '/app/static'
 
-SEND_SUSPICIOUS_BEHAVIOR_WARNINGS = os.getenv('SEND_SUSPICIOUS_BEHAVIOR_WARNINGS')
-
 DEFAULT_FROM_EMAIL = os.getenv('FROM_EMAIL')
 
 EMAIL_BACKEND = 'django.core.mail.backends.dummy.EmailBackend'
```
Comment (markwooff) [inline docker/config.py]: Hmm I mentioned this in the last PR but this line would need to be removed for production. Suggested having it be a conditional that is only set to `True` if the other email envs like `EMAIL_HOST` weren't set

Comment (bryan-robitaille) [reply docker/config.py]: So in this commit it is being removed from the code completely as it’s value is no longer used.  In the series of pull requests I’d like to see the majority of the explicit settings in config.py moved to be controlled through the admin interface.

Comment (bryan-robitaille) [reply docker/config.py]: When moving those settings to be controlled from the interface we can set the logic required in the backend.

Comment (markwooff) [reply docker/config.py]: Where are the SMTP email envs now being set then?

Comment (bryan-robitaille) [reply docker/config.py]: My bad.  MIsread your comment.  The smtp line variables aren’t touched in this PR.  My working branch has the dummy set but the only line removed is the Boolean for the suspicious flag.  GitHub only shows it here for situational context in the file.  While I made the change in my working branch that particular modification was not committed.  So when merged with master the prod setting will not be changed.

Comment (bryan-robitaille) [reply docker/config.py]: Ok.. I finally see what you mean.  In your comment to Nic’s pull request last week you saw the change done in his pull request but the line wasn’t changed before merging.  I’ll fix and add to this PR.

Comment (markwooff) [reply docker/config.py]: Yeah sorry for the confusion @bryan-robitaille - but thanks for removing. Ilia will be using the fixed master branch now for k8s

**Pull request merged**